### PR TITLE
Uninspiration.

### PR DESCRIPTION
**Pull request by pbstFusion**

### CODE REVIEW

**Comment by pbstFusion** [comment]
Truly uninspirational.

File: Server/Server-Uninspiration.lua [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		AdminLevel = "Players";	    -- Admin level; If using settings.CustomRanks set this to the custom rank name (eg. "Baristas")
+		Function = function(plr)    -- Function to run for command
+			local Quotes = {
+				"It's never too late to go back to bed.";
+				"I'm not superstitious, but I'm a little stitious.";
+				"If you hate yourself, find an opinion you disagree with. You'll hate that much more.";
+				"If you keep following your dreams, they'll file a restraining order.";
+				"Don't trust random quotes you see online just because they've got a name. - Lenin";
+				"what even am i doing";
+				"The first step on the road to faliure is trying.";
```
Comment (Expertcoderz) [inline Server/Server-Uninspiration.lua:23]: *failure

**Comment by pbstFusion** [comment]
Boom fixed

**Comment by pbstFusion** [comment]
when the

**Comment by pbstFusion** [comment]
Hurr up!!¡

**Comment by pbstFusion** [comment]
wow what a scam

**Review by LawMixer — APPROVED** [review]
Seems good to me

**Pull request merged**